docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+};
+Openlaw.renderForPreview(agreements[0].agreement, hiddenVariables, paragraphs);
+```
+
+**Response**
+
+Returns the HTML contents of the agreement as a string.
+
+Example
+
+```html
+<div class='openlaw-paragraph paragraph-1'><p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p></div><ul class='list-lvl-1'><li><div class='openlaw-paragraph paragraph-2'><p>1.  <strong>Services</strong>. Advisor agrees to consult with and advise Company from time to time, at Company's request (the "Services"). </p></div></li>
+```
+
+### parseMarkdown
+
+Parse selected paragraph text to edit (in draft mode) to HTML.
+
+```scala
+parseMarkdown(
+  str: String
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `str` | `String` | **Required.** The text of the selected paragraph to edit. |
+
+Example
+
+```js
+const currentParagraph = 'This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:';
+Openlaw.parseMarkdown(currentParagraph);
+```
+
+**Response**
+
+Returns selected paragraph text parsed to HTML.
+
+Example
+
+```html
+<p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p>
+```
+
+### renderParagraphForEdit
+
+Render paragraph of agreement for editing.
+
+```scala
+renderParagraphForEdit(
+  agreement: StructuredAgreement,
+  index: Int
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `agreement` | [`StructuredAgreement`](#structuredagreement) | **Required.** A [`StructuredAgreement` object](#structuredagreement) representing the agreement to be rendered. |
+| `index` | `Int` | **Required.** The index number of the paragraph to be rendered for edit. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const agreements = Openlaw.getAgreements(executionResult.executionResult);
+Openlaw.renderParagraphForEdit(agreements[0].agreement, 1);
+```
+
+**Response**
+
+Returns the HTML contents of the paragraph to be rendered for edit as a string.
+
+Example
+
+```html
+<p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p>
+```
+
+### getTypes
+
+Get all [markup language](/markup-language/) variable types for use in markdown editor.
```

Comment:
Is it confusing how we are using the terms markup vs markdown? Is there any inconsistency there?